Jenkinsfile

```diff
@@ -24,8 +24,8 @@ def buildImages = { ->
       imageNamePy2 = "${imageNameBase}:py2-${gitCommit()}"
       imageNamePy3 = "${imageNameBase}:py3-${gitCommit()}"
 
-      buildImage(imageNamePy2, ".", "py2.7")
-      buildImage(imageNamePy3, "-f Dockerfile-py3 .", "py3.6")
```

Comment:
Should the actual `Dockerfile-py3` be removed?

Reply:
I'll do some larger Dockerfile cleanup in a separate PR. :)